packages/labs/ssr-client/package.json

```diff
@@ -106,9 +106,10 @@
     "@lit-internal/scripts": "^1.0.0"
   },
   "dependencies": {
+    "@lit-labs/ssr-dom-shim": "^1.1.0",
```

Comment:
Should you remove this dependency from lit-element?

Reply:
Will remove when deleting we don't reference the const from it for internals attributes.